airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/services.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Use `number` as type instead of `string`

Reply:
Sample response from the API, trimmed of sensitive content:

```
{
    "totalCount": 7,
    "data": [
       ...
        {
            "id": "6d458cb4-805e-4dbf-a990-1f735ec9f1d6",
            "name": "test",
            "description": "",
            "teamId": "5d68d83f-4237-4329-a565-91d5606c7823",
            "tags": [],
            "links": {
                "web": "https://zip.app.opsgenie.com/service/6d458cb4-805e-4dbf-a990-1f735ec9f1d6/status",
                "api": "https://api.opsgenie.com/v1/services/6d458cb4-805e-4dbf-a990-1f735ec9f1d6"
            },
            "isExternal": false
        },
        ...
    ],
    "paging": {
        "first": "https://api.opsgenie.com/v1/services?limit=20&sort=name&offset=0&order=desc",
        "last": "https://api.opsgenie.com/v1/services?limit=20&sort=name&offset=0&order=desc"
    },
    "took": 0.036,
    "requestId": "c1751dd0-6ca7-4886-8011-18ee1036e262"
}
```